specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -1228,6 +1228,283 @@
+      "get": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Get",
+        "description": "Get the detail of a linked workspace.",
+        "x-ms-examples": {
+          "GetLinkedWorkspace": {
+            "$ref": "./examples/getLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the detail of the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "404": {
+            "description": "Specific linked workspace not found.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Update",
+        "description": "Update the detail of a linked workspace.",
+        "x-ms-examples": {
+          "UpdateLinkedWorkspace": {
+            "$ref": "./examples/updateLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The object for updating an existing linked workspace",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspaceUpdateDto"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the detail of the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "404": {
+            "description": "Specific linked workspace not found.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          },
```

Comment:
The swagger convention is not to list errors explicitly but just rely on the `default` response to capture them all. Explicitly declaring a non-default error also impacts the generated SDK.

Reply:
will remove it.